app/javascript/fractalsoft/phone.js

```diff
@@ -0,0 +1,26 @@
+import intlTelInput from 'intl-tel-input';
+
+document.addEventListener('turbolinks:load', function() {
+  var input_phone = document.querySelector('.js-phone-input');
+  var input_country_code = document.querySelector('.js-country-code');
+  var input_country_iso2 = document.querySelector('.js-country-iso2');
+
+  var iti = intlTelInput(input_phone, {
+    separateDialCode: true,
+    initialCountry: "auto",
+    geoIpLookup: function(callback) {
+      fetch("https://ipapi.co/json")
+        .then(function(res) { return res.json(); })
+        .then(function(data) { callback(data.country_code); })
+        .catch(function() { callback("us"); });
+    }
+  });
+
+  input_country_code.value = iti.getSelectedCountryData().dialCode;
+  input_country_iso2.value = iti.getSelectedCountryData().iso2;
+
+  input_phone.addEventListener('countrychange', function(e) {
+  input_country_code.value = iti.getSelectedCountryData().dialCode;
+  input_country_iso2.value = iti.getSelectedCountryData().iso2;
+  });
```

Comment:
I think you've lost the indentation here.
```suggestion
  input_phone.addEventListener('countrychange', function(e) {
    input_country_code.value = iti.getSelectedCountryData().dialCode;
    input_country_iso2.value = iti.getSelectedCountryData().iso2;
  });
```